src/main/kotlin/com/kafkakotlin/demo/kafka/producer/KafkaProducer.kt

```diff
@@ -0,0 +1,17 @@
+package com.kafkakotlin.demo.kafka.producer
+
+import com.kafkakotlin.demo.users.User
+import java.util.UUID
+import org.springframework.kafka.core.KafkaTemplate
+import org.springframework.stereotype.Component
+
+@Component
+class KafkaProducer(
+    private val kafkaTemplate: KafkaTemplate<String, User>
+) {
+
+    fun strikeMessageToKafka(payload: User) {
```

Comment:
Interesting method naming

Reply:
As in the name doesn't make sense? Im not the best with naming...

Reply:
Sounds very military haha, i'd just call it `publish`